Plash/OpenURLView.swift

```diff
@@ -28,7 +28,7 @@ struct OpenURLView: View {
 					HStack(spacing: 3) {
 						Text("You could, for example,")
 						Button("show the time.") {
-							self.urlString = "https://time.pablopunk.now.sh"
+							self.urlString = "https://time.pablopunk.com"
```

Comment:
```suggestion
							self.urlString = "https://time.pablopunk.com/?seconds&fg=white&bg=transparent"
```

Let's make it nicer.